docs/HowToFAQ/FAQModuleImport.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Why is my import opened statement not working?
+---
+
+## Question
+
+Why is my import opened statement not working in the following example:
+```
+{% include_relative FAQModuleImport.dfy %}
+```
+
+## Answer
+
+Although nested modules are nested inside other modules, they should really be thought of as their own module.
+There is no particular benefit to module `A.B` from being nested inside module `A` --- it does not see any declarations of `A` other than sibling modules of `B`.
+The benefit is to module `A`, for which the nested module `B` is a namespace that can group some related declarations together.
```

Comment:
I don't understand this part

Reply:
I'm not sure of your confusion. For an enclosing module A, a nested module A.B serves to group related declarations within B, taking them out of A's direct namespace, but leaving them easily accessible (without needing an import) within A.

Reply:
I just meant that the text was not clear to me (I don't know what it means for a module to "see" another module, and I'm not sure that a module counts as a "declaration")

Reply:
Revised